tests/test_authentication/test_authentication.py

```diff
@@ -11,6 +11,37 @@
 
 class TestAuthentication(BaseTestCase):
     def test_invalid_token_returns_error_message(self):
+        """
+        Test that an error message is returned when a user
+        has an invalid token
+        """
+        headers = {"Authorization": "Bearer" + " " + "INVALID_TOKEN"}  # noqa E501
+        response = self.app_test.post(
+            '/mrm?query=' + office_mutation_query,
+            headers=headers)
+        self.assertIn("Invalid token. Please Provide a valid token!", str(response.data)) # noqa E501
```

Comment:
These statements can be modified without using the noqa

Reply:
@Bonifase I have done that